src/Types/Msg.hs

```diff
@@ -6,6 +6,7 @@ data Msg = Msg
   { subject   :: ByteString,
     sid       :: ByteString,
     replyTo   :: Maybe ByteString,
-    byteCount :: Int,
-    payload   :: Maybe ByteString
+    byteCount :: Int, -- TODO: not sure we need to store this
```

Comment:
I don't think this needs storing. It's just used on the parse to read the correct amount of chars